Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -69,9 +104,52 @@ public final class ProjectPamphletViewController: UIViewController {
     }
   }
 
+  private func setupConstraints() {
+    // swiftlint:disable line_length
+    let backThisProjectContainerViewConstraints = [self.backThisProjectContainerView.leftAnchor.constraint(equalTo: self.view.leftAnchor),
+                                                   self.backThisProjectContainerView.rightAnchor.constraint(equalTo: self.view.rightAnchor),
+                                                   self.backThisProjectContainerView.bottomAnchor.constraint(equalTo: self.view.bottomAnchor)
+    ]
+    let containerMargins = self.backThisProjectContainerView.layoutMarginsGuide
+
+    let backThisProjectButtonConstraints = [self.backThisProjectButton.leftAnchor.constraint(equalTo: containerMargins.leftAnchor),
+                                            self.backThisProjectButton.rightAnchor.constraint(equalTo: containerMargins.rightAnchor),
+                                            self.backThisProjectButton.bottomAnchor.constraint(equalTo: containerMargins.bottomAnchor),
+                                            self.backThisProjectButton.topAnchor.constraint(equalTo: containerMargins.topAnchor),
+                                            self.backThisProjectButton.heightAnchor.constraint(equalToConstant: Styles.buttonMinHeight)
+    ]
+
+    NSLayoutConstraint.activate([backThisProjectContainerViewConstraints,
+                                 backThisProjectButtonConstraints].flatMap { $0 })
+  }
+
+  public override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.backThisProjectContainerView
+      |> \.layoutMargins .~ .init(all: backThisProjectContainerViewMargins)
+
+    _ = self.backThisProjectButton
+      |> greenCTAButtonStyle
+      |> UIButton.lens.title(for: .normal) %~ { _ in
+        return Strings.project_back_button()
+    }
+
+    _ = self.backThisProjectButton.titleLabel
+      ?|> \.font .~ UIFont.boldSystemFont(ofSize: 15)
```

Comment:
Is there a reason we're not using ksr font? Do we not want to support Dynamic Type?

Reply:
Yeah I figured we don't want to support dynamic type on this button, similar to how we don't support dynamic type on navigation items. What do you think?

Reply:
Currently most of our buttons support Dynamic Type (even though not great) - mostly because we use UIButton subclasses. Would it be too hard to implement this in the same manner as we do now? (please see the screenshot)

<img width="538" alt="Screen Shot 2019-04-12 at 3 48 12 PM" src="https://user-images.githubusercontent.com/387596/56070325-1566f280-5d3c-11e9-8afd-ec91070b6bde.png">

Reply:
Yep can do! 👍

Reply:
I've noticed that the button now supports Dynamic Type (resizes the font)...but still cuts of the text in the largest setting. What do you think about making the button support multiple lines and allowing it to display the whole text?

To clarify, this is what's happening currently:
![IMG_0123](https://user-images.githubusercontent.com/387596/56158238-6530fe00-5f76-11e9-85dc-763c3de98c05.PNG)

Adding a simple implementation of a custom button (that could potentially fix our other buttons (in the above screenshot) like so can fix some of those problems:

```
// Could probably name it better but I was just quickly trying to get it working
class CustomButton: UIButton {
  override var intrinsicContentSize: CGSize {
    let size = self.titleLabel?.intrinsicContentSize ?? .zero
    let titleInsets = self.titleEdgeInsets

    return CGSize(
      width: titleInsets.left + size.width  + titleInsets.right,
      height: titleInsets.top + size.height + titleInsets.bottom
    )
  }

  override func layoutSubviews() {
    super.layoutSubviews()

    self.titleLabel?.preferredMaxLayoutWidth = self.titleLabel?.frame.width ?? 0
  }
}
```

Which fixes the button and allows multiline text if configured like so
```
self.backThisProjectButton.titleLabel?.numberOfLines = 0 // could be in the styling function
```

we would of course have to replace button's height constraint to allow for `greaterThan` as oppose to equal

```
self.backThisProjectButton.heightAnchor.constraint(greaterThanOrEqualToConstant: Styles.minTouchSize.height)
```

![IMG_0125](https://user-images.githubusercontent.com/387596/56158356-a75a3f80-5f76-11e9-9fe7-907dd39aa651.PNG)

We can also go one step further and make the spacing slightly better:
```
self.backThisProjectButton.titleEdgeInsets = UIEdgeInsets(topBottom: Styles.grid(2)) // could be in the styling function
```

![IMG_0126](https://user-images.githubusercontent.com/387596/56158432-d83a7480-5f76-11e9-804a-d42baac584e6.PNG)

And finally we would have to fix the table view's content insets to accommodate for the button properly (this screenshot demonstrates scroll view scrolled all the way to the bottom - you can see how it's cutting off the last row)

![IMG_0124](https://user-images.githubusercontent.com/387596/56158537-2780a500-5f77-11e9-8b88-6d8faf2810df.PNG)

Reply:
Done